build.sbt

```diff
@@ -1119,6 +1128,7 @@ lazy val `dev-environment` = (project in file("dev"))
 
 lazy val `reloadable-server` = (project in file("dev") / "reloadable-server")
   .settings(runtimeLibCommon: _*)
+  .settings(noMima)
```

Comment:
This is a published artifact. Why disable MiMa here?

Reply:
Because it's a dev mode / build tool support artifact, same with the build-tool-support below.